src/@types/types.tsx

```diff
@@ -0,0 +1,10 @@
+export interface IRegisterFormProperties {
```

Comment:
I think we can keep these right next to their corresponding functions, `@types` should ideally be kept for third party types that are not available or things like declarations.

These two also seem to be either misnamed or misused. It seems like they're being used for state where as being called `Properties`. It's bike shedding really, however it's something that may come back later and confuse the future us whilst debugging code.